app/api/v2/handlers/operation_api.py

```diff
@@ -122,7 +162,7 @@ async def get_operation_link_result(self, request: web.Request):
         return web.json_response(result)
 
     @aiohttp_apispec.docs(tags=['operations'])
-    @aiohttp_apispec.request_schema(LinkSchema(partial=True, only=['command', 'status']))
+    @aiohttp_apispec.request_schema(LinkSchema(partial=True))
```

Comment:
I don't think that these are part of your ticket, but could you add back `only=['command', 'status']` just for this endpoint?